recipes/cassandra-cpp-driver/all/conanfile.py

```diff
@@ -0,0 +1,145 @@
+import os.path
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class CassandraCppDriverConan(ConanFile):
+    name = "cassandra-cpp-driver"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://docs.datastax.com/en/developer/cpp-driver/"
+    description = "DataStax C/C++ Driver for Apache Cassandra and DataStax Products"
+    topics = ("cassandra", "cpp-driver", "database", "conan-recipe")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "install_header_in_subdir": [True, False],
+        "use_atomic": [None, "boost", "std"],
+        "with_openssl": [True, False],
```

Comment:
Is openssl the only tls provider?

Reply:
Yep openssl is the only tls provider in this package.